ci-operator/step-registry/hypershift/aws/create/hypershift-aws-create-chain.yaml

```diff
@@ -174,11 +207,6 @@ chain:
       </testsuite>
       EOF
       fi
-      set -e
-
-      echo "Cluster became available, creating kubeconfig"
```

Comment:
It's there a reason to remove this?

Reply:
It is not removed. It's just moved to an earlier place, right after the hosted cluster becomes "Available".
This is to make it work with HYPERSHIFT_NETWORK_TYPE==Other where the cluster doesn't get to "Completed" state yet. In that case, we need the kubeconfig for checking Nodes in the hosted cluster and whether they match the desired number in HostedCluster spec.
The cluster get "Completed" only later after installing the Cilium network stack.